### PR TITLE
Prepare initial third party services

### PR DESCRIPTION
**Pull request by Fryguy**
Prepare initial third party services

### CODE REVIEW

**Comment by Fryguy** [comment]
@jrafanie @NickLaMuro Can one of you merge?

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
-sudo: false
 language: ruby
 cache: bundler
 rvm:
-  - 2.4.5
-before_install: gem install bundler -v 1.17.1
+- 2.4.5
+- 2.5.3
+before_install:
+- 'echo ''gem: --no-ri --no-rdoc --no-document'' > ~/.gemrc'
```
Comment (NickLaMuro) [inline .travis.yml]: Any reason this couldn't just be:

```yaml
- 'echo "gem: --no-ri --no-rdoc --no-document" > ~/.gemrc'
```

The double single quotes kinda weird me out (personal problem most likely...)

File: spec/bundler/inject_spec.rb [hunk 2]
```diff
@@ -2,8 +2,4 @@
   it "has a version number" do
     expect(Bundler::Inject::VERSION).not_to be nil
   end
-
-  it "does something useful" do
```
Comment (NickLaMuro) [inline spec/bundler/inject_spec.rb]: Clearly, this was not useful...

File: .rubocop_local.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+AllCops:
+  Exclude:
+  - db/schema.rb
```
Comment (NickLaMuro) [inline .rubocop_local.yml]: Uh, does this make sense having this here?

File: .travis.yml (hunk 1 above)
Comment (Fryguy) [reply .travis.yml]: yeah, that's how YAML.dump dumps it, so I'm gonna leave it

File: .rubocop_local.yml (hunk 3 above)
Comment (Fryguy) [reply .rubocop_local.yml]: probably not...I must have that in my tool that way :/

**Comment by Fryguy** [comment]
@NickLaMuro Updated

**Pull request merged**